WordPress/src/main/java/org/wordpress/android/ui/mysite/ImprovedMySiteFragment.kt

```diff
@@ -540,4 +547,15 @@ class ImprovedMySiteFragment : Fragment(R.layout.new_my_site_fragment),
     override fun onNeutralClicked(instanceTag: String) {
         Toast.makeText(context, "QS - Neutral Clicked", Toast.LENGTH_LONG).show()
     }
+
+    override fun onConfirm(result: Bundle?) {
+        if (result != null) {
+            val task = result.getSerializable(QuickStartFullScreenDialogFragment.RESULT_TASK) as? QuickStartTask
+            viewModel.onQuickStartFullScreenDialogConfirm(task)
+        }
+    }
```

Comment:
Suggestion (💡): What if the above was replaced with the below:
```
    override fun onConfirm(result: Bundle?) {
        viewModel.onQuickStartFullScreenDialogConfirm(
                task = result?.getSerializable(QuickStartFullScreenDialogFragment.RESULT_TASK) as? QuickStartTask
        )
    }
```

Reply:
In 16fd588